cmd/tx/fund_faucet/fund_faucet.go

```diff
@@ -9,12 +9,22 @@ import (
 	"github.com/dymensionxyz/roller/config"
 	"github.com/dymensionxyz/roller/relayer"
 	"github.com/spf13/cobra"
+	"math/big"
 	"os/exec"
 	"path/filepath"
 )
 
+var flagNames = struct {
+	tokenAmount string
+}{
+	tokenAmount: "token-amount",
+}
+
+const faucetAddr = "dym1g8sf7w4cz5gtupa6y62h3q6a4gjv37pgefnpt5"
+const faucetDefaultTokenAmount = "5000000"
```

Comment:
it supposed to be 500 if u multiply it by decimals, no?

Reply:
The [docs](https://docs.dymension.xyz/build/quick-start/roller-quick/ibc-transfer/) defaults to 5,000,000 with 18 zeros. WDYT @shaolin-flow